docs/index.md

```diff
@@ -9,24 +9,73 @@ permalink: index.html
 
 ## Overview
 
-This plugin's main focus is to build [pipeline](https://jenkins.io/doc/book/pipeline/) steps so that we make DevOps process smoother with [Jenkins](https://jenkins.io/).
+Jira-steps-plugin extends [pipeline DSL](https://jenkins.io/doc/book/pipeline/syntax/) by implementing steps for JIRA.
+This plugin generates pipeline script for various JIRA steps. Instead of writing non readable lines of shell script, this plugin
+plugin provides a simple block of code in groovy that can talk to JIRA API. This block of code can be added to the pipeline script in your pipeline job configuration. This makes your DevOps automation Process management fast and reliable.
 
-## Features
+1. JIRA steps will be available in **jenkins > your-pipeline-job > pipeline syntax > snippet generator** after [installing](https://jenkinsci.github.io/jira-steps-plugin/install.html) this plugin.
+2. Required JIRA step can be selected from the sample step drop down.
+3. Pipeline script for the selected JIRA step can be generated by providing the required input fields.
+4. The generated pipeline script can be used
+    * in configuring your pipeline job or
+    * added as a step in your project's [jenkins file](https://jenkins.io/doc/book/pipeline/jenkinsfile/).
 
-We started of with following JIRA Software steps, will keep enhancing it to support other part of API too.
+## JIRA Steps
 
-* Component.
-* Version.
-* Issue.
-* Comments.
-* Watchers.
-* Transitions.
-* IssueLink.
-* Project.
-* Search.
+We started with the following JIRA steps, will keep enhancing it to support other JIRA administration steps.
+
+* Comments
+  * [Get Comments](https://jenkinsci.github.io/jira-steps-plugin/jira_get_comments.html)
```

Comment:
I felt this list is kind of lengthy, instead can we refer to the pipeline steps. 

It would be nice we can have 
* High level overview 
* Couple of examples with some detailed explanation, which should cover the usage of each plugin on high level.